q2_fmt/plugin_setup.py

```diff
@@ -123,7 +128,10 @@
                           ' grouped diversity data and selected comparison.',
     },
     name='Engraftment Pipeline for FMT Analysis',
-    description='',
+    description='Applies group timepoints to Fecal Microbiota Transplant data'
+         ' and then applies statistical tests(Wilcoxon rank sum test or'
```

Comment:
```suggestion
         ' and then applies statistical tests (Wilcoxon rank sum test or'
```

Reply:
Oh, it's actually the `Wilcoxon _signed rank test_` (the `rank sum test` is also known as the `Wilcoxon-Mann-Whitney`/`Mann-Whitney-U` test)

Reply:
Roughly, Wilcoxon described both of these tests at once, and Mann and Whitney created a more useful derivation of the latter test, so they tend to get credited more often for the rank-sum test.